### PR TITLE
Module exports to allow importing

### PR DESCRIPTION
**Pull request by leodev9**
### What did you change/add/fix?

Update yml commands and package.json.

### What testing you did?

None

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -7,12 +7,23 @@ jobs:
...
     steps:
       - uses: actions/checkout@v3
-      # Setup .npmrc file to publish to npm
       - uses: actions/setup-node@v3
         with:
-          node-version: '16.x'
-          registry-url: 'https://registry.npmjs.org'
+          node-version: 16
+      - run: npm i
+      - run: npm test
```
Comment (reinismu) [inline .github/workflows/publish.yml:14]: I wouldn't run test again. In this job just focus on publishing.

File: package.json [hunk 2]
```diff
@@ -8,11 +8,12 @@
     "dist"
   ],
   "scripts": {
-    "build": "rm ./dist/* && tsc",
+    "build": "tsc",
     "tsc": "tsc --skipLibCheck",
-    "test": "jest",
-    "lint": "eslint . --ext .ts",
-    "lint:fix": "eslint . --fix --ext .ts"
+    "format": "prettier --write \"src/**/*.ts\"",
```
Comment (reinismu) [inline package.json:13]: Our `eslint` should take care of formatting

File: package.json [hunk 3]
```diff
@@ -8,11 +8,12 @@
     "dist"
   ],
   "scripts": {
-    "build": "rm ./dist/* && tsc",
+    "build": "tsc",
     "tsc": "tsc --skipLibCheck",
```
Comment (reinismu) [inline package.json:12]: Do we need `tsc` script then if we have `build` that does the same thing?

File: .github/workflows/publish.yml [hunk 4]
```diff
@@ -7,12 +7,22 @@ jobs:
...
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
       - run: npm ci
+      - run: npm run build
       - run: npm publish --access public
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```
Comment (reinismu) [inline .github/workflows/publish.yml:28]: Why 2 jobs? I think there should be just 1 where we build and publish

Comment (leodev9) [reply .github/workflows/publish.yml:28]: Let's try putting in 1 job

**Review by reinismu — APPROVED** [review]

**Pull request merged**